src/Text/Pandoc/Writers/Docx.hs

```diff
@@ -975,14 +975,11 @@ listItemToOpenXML opts numid (first:rest) = do
         _             -> False
   -- Prepend an empty string if the first entry is another
   -- list. Otherwise the outer bullet will disappear.
-  let (first', rest') = if isListBlock first
-                           then (Plain [Str ""] , first:rest)
-                           else (first, rest)
-  first'' <- withNumId numid $ blockToOpenXML opts first'
-  -- baseListId is the code for no list marker:
-  rest''  <- withNumId baseListId $ blocksToOpenXML opts rest'
+  let bs' = if isListBlock (head bs) then (Plain [Str ""]:bs) else bs
```

Comment:
We try to avoid using unsafe partial functions like `head`.
You can use `headMay` from `Safe` or use pattern matching (e.g. a case statement).